src/Acuminator/Acuminator.Analyzers/Analyzers/DAC/DacPropertyAttributes/DacExtensionDefaultAttributeAnalyzer.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Acuminator.Utilities;
+using System.Collections.Immutable;
+using PX.Data;
+
+namespace Acuminator.Analyzers
+{
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	class DacExtensionDefaultAttributeAnalyzer : PXDiagnosticAnalyzer
+	{
+
+		private const string _PersistingCheck = "PersistingCheck";
+		
+		public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics =>
+			ImmutableArray.Create
+			(
+				Descriptors.PX1030_DefaultAttibuteToExisitingRecords
+			);
+#pragma warning disable CS4014
+		internal override void AnalyzeCompilation(CompilationStartAnalysisContext compilationStartContext, PXContext pxContext)
+		{
+			compilationStartContext.RegisterSymbolAction(symbolContext =>
```

Comment:
`async symbolContext => await AnalyzePropertyAsync`

Reply:
+1, I think I misused it too in DAC property / attributes type mismatch diagnostic so I'll fix it there